src/onesignal/OneSignal.ts

```diff
@@ -105,7 +105,7 @@ export default class OneSignal {
       throw new InvalidArgumentError('externalId', InvalidArgumentReason.WrongType);
     }
 
-    if (typeof jwtToken !== 'string') {
+    if (!!jwtToken && typeof jwtToken !== 'string') {
```

Comment:
Thinking we should check `undefined` here instead of using `!!`, because `false` is not going to throw the error when it should.